features/subtraction.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Subtraction
```

Comment:
No test for subtracting a number from itself, which was in the acceptance criteria.